src/CookieStore.ts

```diff
@@ -128,6 +142,22 @@ class CookieStore {
 
     localStorage.setItem(PERSISTENCY_KEY, JSON.stringify(serializedCookies))
   }
+
+  _deleteExpiredCookies() {
```

Comment:
Let's mark this method as [`private`](https://www.typescriptlang.org/docs/handbook/classes.html#understanding-typescripts-private), which would allow us to remove the leading dangle. 

Marking the method as private will forbid its public access, unless it's used within the class declaration.

Reply:
Ups sorry. For some reason I was in JavaScript-mode when writing this. I fixed it.